qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/BlockedSearcherRestCancellationTestCase.java

```diff
@@ -57,6 +61,8 @@
  */
 public abstract class BlockedSearcherRestCancellationTestCase extends HttpSmokeTestCase {
 
+    private static final Logger testCaseLogger = LogManager.getLogger(BlockedSearcherRestCancellationTestCase.class);
```

Comment:
[opt] ESTestCase is also a test case. Could say blockedTestCaseLogger, to clarify it's this test case specifically. Or a comment saying you're deviating from the norm because there's already a `logger` -- I wondered why and went to look.

Reply:
It's really because `logger` means `Engine#logger` in the places we add logging here, and that's pretty noisy.